client/state/data-layer/wpcom/read/following/mine/test/index.js

```diff
@@ -151,14 +152,14 @@ describe( 'get follow subscriptions', () => {
 					blog_ID: 122463145,
 					URL: 'http://readerpostcards.wordpress.com',
 					feed_URL: 'http://readerpostcards.wordpress.com',
-					date_subscribed: Date.parse( '2017-01-12T03:55:45+00:00' ),
+					date_subscribed: moment( '2017-01-12T03:55:45+00:00' ),
```

Comment:
Just being able to read the date without doing mental gymnastics is nice. Not critical though.